includes/Core/Authentication/Authentication.php

```diff
@@ -660,8 +660,11 @@ private function handle_oauth() {
 	 * @return array Filtered $data.
 	 */
 	private function inline_js_base_data( $data ) {
+		global $wp_version;
+
 		$data['isOwner']             = $this->owner_id->get() === get_current_user_id();
 		$data['isFirstAdmin']        = $data['isOwner'] || ( ! $this->owner_id->get() && current_user_can( Permissions::MANAGE_OPTIONS ) );
+		$data['isWP5.0+']            = version_compare( $wp_version, '5.0.0', '>=' );
```

Comment:
This is a bit unnecessarily specific. Why not expose the version to the client in a way that's just as easy to use but more flexible? We do something similar in E2E as it is where one of the utilities requires WP5 as well. I think using a similar strategy here would be just as easy and more valuable. See https://github.com/google/site-kit-wp/blob/aa6cdd0792898920dd626dd71b2d722663a50756/tests/e2e/mu-plugins/e2e-rest-core-version.php#L27-L34

Perhaps the key could be `wpVersion` with a shape like `{ full, major, minor }` ?

Then we would only need to check `wpVersion.major < 5` 😄 

Also note we don't need to touch the `$wp_version` global to get the same value.

Reply:
Yep, i like it. Updated.